pandas/tseries/tests/test_base.py

```diff
@@ -154,6 +155,86 @@ def test_minmax(self):
             obj = DatetimeIndex([pd.NaT, pd.NaT, pd.NaT])
             self.assertTrue(pd.isnull(getattr(obj, op)()))
 
+    def test_numpy_minmax(self):
+        dr = pd.date_range(start='2016-01-15', end='2016-01-20')
+        self.assertEqual(np.min(dr), Timestamp(
+            '2016-01-15 00:00:00', offset='D'))
+        self.assertEqual(np.max(dr), Timestamp(
+            '2016-01-20 00:00:00', offset='D'))
+
+        errmsg = "the 'out' parameter is not supported"
+        tm.assertRaisesRegexp(ValueError, errmsg, np.min, dr, out=0)
+        tm.assertRaisesRegexp(ValueError, errmsg, np.max, dr, out=0)
+
+        self.assertEqual(np.argmin(dr), 0)
+        self.assertEqual(np.argmax(dr), 5)
+
+        if not _np_version_under1p10:
+            errmsg = "the 'out' parameter is not supported"
+            tm.assertRaisesRegexp(ValueError, errmsg, np.argmin, dr, out=0)
+            tm.assertRaisesRegexp(ValueError, errmsg, np.argmax, dr, out=0)
+
+    def test_round(self):
```

Comment:
do we need to test floor/ceil? (as they are numpy methods as well) (and now on the datetimelike indexes)